test/unit/lib/static/components/controls/browser-list/index.jsx

```diff
@@ -1,26 +1,39 @@
+import {expect} from 'chai';
 import React from 'react';
 import BrowserList from '../../../../../../../lib/static/components/controls/browser-list';
 import {ThemeProvider} from '@gravity-ui/uikit';
+import {render} from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
 
 describe('<BrowserList />', () => {
     const sandbox = sinon.sandbox.create();
 
+    beforeEach(() => {
+        global.MutationObserver = class {
+            constructor() {}
+            disconnect() {}
+            observe() {}
+        };
+    });
+
     afterEach(() => sandbox.restore());
```

Comment:
Lets remove `global.MutationObserver` in `afterEach` in order to not pollute `global`